p2pd/main.go

```diff
@@ -360,21 +364,6 @@ func main() {
 		log.Fatal(err)
 	}
 
-	if c.AutoNat {
-		var opts []libp2p.Option
-		// allow the AutoNAT service to dial back quic addrs.
-		if c.QUIC {
-			opts = append(opts,
-				libp2p.DefaultTransports,
-				libp2p.Transport(quic.NewTransport),
```

Comment:
what about quic / where have we refactored the autonat service to?

Reply:
nevermind - i see that we're using the one in the base go-libp2p with this change.